### PR TITLE
Add action creation test with maximum time limit

### PR DESCRIPTION
**Pull request by sven-lange-last**
## Description
Verify that an action with the maximum supported run time limit can be created successfully. This integration test is a companion to #3379 which introduces configurable action time limits and makes sure that the configured limit actually applies.

## Related issue and scope
Integration test for #3379.

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [X] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [X] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [X] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [X] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

**Comment by sven-lange-last** [comment]
PG 3 / 2008 running.

File: tests/dat/actions/timeoutLean.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * of milliseconds before returning. Uses a timer instead of a busy loop.
+ * The function actually waits slightly longer than requested.
+ *
+ * @param parm Object with Number property timeoutInMs
+ * @returns Object with String property msg describing how long the function waited
+ */
+function main(parm) {
+    console.log("Specified timeout is " + parm.timeoutInMs + " ms.")
+
+    return new Promise(function(resolve, reject) {
```
Comment (rabbah) [inline tests/dat/actions/timeoutLean.js]: Isn’t there a timeout action alreaady?

**Comment by rabbah** [comment]
Can we dedup with this action https://github.com/apache/incubator-openwhisk/blob/master/tests/dat/actions/timeout.js

**Comment by sven-lange-last** [comment]
@rabbah 
* There is no need to exclude this test in Travis because it only tries to *create* the action. It does *not execute* it. So the test is probably much faster than most of the other action limit tests.
 * You are right, there already is a [timeout action](https://github.com/apache/incubator-openwhisk/blob/master/tests/dat/actions/timeout.js). This timeout action uses a busy wait loop and thus, uses much more CPU than a timeout should use. My plan is to replace the existing timeout action with my new lean timeout action in a following PR once this PR is merged.

**Comment by rabbah** [comment]
Great. Thanks got the clarifications.

**Comment by rabbah** [comment]
I’d suggest moving the test to the basic usage suite.

**Comment by rabbah** [comment]
Hmm actually... maybe this can just be a Actions API Unit test.

**Comment by rabbah** [comment]
Also we have these tests https://github.com/apache/incubator-openwhisk/blob/a8434b7c3eccaa5b4627849ce3cbff78dc67b41e/tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala#L705.

**Comment by sven-lange-last** [comment]
Superseded by #3485. Closing this PR.
